sunkit_magex/pfss/utils.py

```diff
@@ -79,6 +81,7 @@ def fix_hmi_meta(hmi_map):
         hmi_map.meta.update(_earth_obs_coord_meta(hmi_map.meta['date-obs']))
 
 
+@deprecated('1.0', message="This is not required anymore. An example of how to load an ADAPT file is available in the documentation.")
```

Comment:
I'd either link to it or say use `sunpy.map.Map` or similar.

Reply:
Done